src/pages/products/Products.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { FaTrashAlt, FaBatteryThreeQuarters, FaCheck, FaEdit, FaShoppingBag } from 'react-icons/fa';
+
+import { getProductsByCategory } from '../../services';
+import InputSpan from '../../components/form';
+import * as Constants from '../../constants';
+
+import './products.scss';
+
+class Products extends Component {
+
+    state = {
+        products: null,
+        displayFilter: []
+    };
+
+    insideRefs = [];
+
+    setRef = (numberOfRefs) => {
+        this.insideRefs = [...Array(numberOfRefs)]
+            .map((inputSpan) => inputSpan = React.createRef())
+    };
+
+    componentDidMount() {
+        getProductsByCategory()
+            .then(products => {
+                this.setState({ products: products.products });
+            })
+    }
+
+    deleteTask = () => {
+        console.log('deleted')
+    }
+
+    simulateClick(index) {
+        this.insideRefs[index].current.handleSpanBlur();
```

Comment:
why `blur` is invoked programmatically?

Reply:
@iliatcymbal I do it because in fact I have native click by another component.

Reply:
@ex3emxl so why this fact forces you to make programmatically blur? It is wrong. Which goal do you try to achieve?